pkg/dto/restore_policy.go

```diff
@@ -10,17 +10,19 @@ import (
 // @Description RestorePolicy represents a policy for the restore operation.
 type RestorePolicy struct {
 	// The number of concurrent record readers from backup files.
+	// This value determines the number of threads the backup service will use.
```

Comment:
```suggestion
	// This value controls the level of parallelism used by the backup service when
        // reading backup files.
```
Threads term is not really applicable in this context.